### PR TITLE
Update link to the Quick Start Guide

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -53,8 +53,7 @@ nav:
   - Home: index.md
   - Get Started:
       - What is Choreo?: get-started/what-is-choreo.md
-      - Quick Start Guide: get-started/quick-start-guide.md
-      - Get Started: get-started/get-started.md
+      - Quick Start Guide: get-started/get-started.md
```
Comment (nilminiwso2) [inline en/mkdocs.yml:56]: Have we applied a page redirect?

Comment (RukshiW) [reply en/mkdocs.yml]: Added the new content to the existing Quick Start Guide itself so that no redirection is needed.

File: en/mkdocs.yml [hunk 2]
```diff
@@ -53,8 +53,7 @@ nav:
   - Home: index.md
   - Get Started:
       - What is Choreo?: get-started/what-is-choreo.md
-      - Quick Start Guide: get-started/quick-start-guide.md
-      - Get Started: get-started/get-started.md
+      - Quick Start Guide: get-started/quick-start-guide
```
Comment (nilminiwso2) [inline en/mkdocs.yml:56]:
```suggestion
      - Quick Start Guide: get-started/quick-start-guide.md
```

File: en/docs/get-started/quick-start-guide.md [hunk 3]
```diff
@@ -1,257 +1,477 @@
-# Quick Start Guide
+# Get Started with Choreo
```
Comment (shaniR) [inline en/docs/get-started/quick-start-guide.md:1]: If we are calling this Get started, then let's have the file name with the same name too.. or change the title to match the file name.

**Review by shaniR — CHANGES_REQUESTED** [review]

**Review by nilminiwso2 — DISMISSED** [review]

**Review by nilminiwso2 — APPROVED** [review]

**Review by shaniR — APPROVED** [review]

**Pull request merged**